migrations/2024-07-31-063531_alter_customer_id_in_payouts/up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE payouts
+ALTER COLUMN customer_id
+DROP NOT NULL,
+ALTER COLUMN address_id
+DROP NOT NULL;
+
+ALTER TABLE payout_attempt
+ALTER COLUMN customer_id
+DROP NOT NULL,
+ALTER COLUMN address_id
+DROP NOT NULL;
```

Comment:
can you add 'IF EXISTS' for these queries

Reply:
IF EXISTS is not supported in such queries